M2/Macaulay2/packages/ReesAlgebra.m2

```diff
@@ -1980,7 +2000,6 @@ doc ///
     [reesAlgebra,Strategy]
     [isLinearType,Strategy]
     [isReduction, Strategy]    	  
-    [normalCone, Strategy]    	  
```

Comment:
You can document the strategy for `associatedGradedRing` here instead, and the for the other options,

Reply:
Done